.github/workflows/appsignals-e2e-ec2-test.yml

```diff
@@ -31,13 +31,17 @@ env:
   LOG_GROUP_NAME: /aws/appsignals/generic
   TEST: ${{ inputs.test }}
   GET_ADOT_JAR_COMMAND: "wget -O adot.jar https://github.com/aws-observability/aws-otel-java-instrumentation/releases/latest/download/aws-opentelemetry-agent.jar"
+  TEST_RESOURCES_FOLDER: /__w/aws-application-signals-test-framework/aws-application-signals-test-framework
```

Comment:
Why are we changing `/home/runner/work` to be `/__w/`? what does `/__w/` mean?

Reply:
That seems to be the default directory of the docker image we are using as the environment